install/configuring_inventory_file.adoc

```diff
@@ -1609,6 +1596,13 @@ persistent storage for logging. If `openshift_logging_es_pvc_dynamic` is
 not set, then cluster logging data is stored in an `emptyDir` volume, which will
 be deleted when the Elasticsearch pod terminates.
 
+[IMPORTANT]
```

Comment:
Like with the verbiage here can likely be more specific (just as we should above).

In short, we should use the notes/warning from: 
https://docs.openshift.com/container-platform/3.10/install_config/aggregate_logging.html#aggregated-elasticsearch
and point to http://lucene.apache.org/core/3_2_0/api/all/org/apache/lucene/index/IndexWriter.html#deletionPolicy 

Suggested wording change:
```
Due to ElasticSearch not implementing a custom deletionPolicy, the use of NFS storage as a volume or a persistent volume is not supported for Elasticsearch storage, as Lucene and the default deletionPolicy, relies on file system behavior that NFS does not supply. Data corruption and other problems can occur.

NFS implementations on the marketplace may not have these issues. Please contact the individual NFS implementation vendor for more information on any testing they may have performed against these OpenShift framework components.
```

Reply:
I added this in with the existing content + minor tweaks in the new PR.